connector/connect/server/src/main/scala/org/apache/spark/sql/connect/execution/ExecuteThreadRunner.scala

```diff
@@ -92,7 +92,8 @@ private[connect] class ExecuteThreadRunner(executeHolder: ExecuteHolder) extends
         "execute",
         executeHolder.responseObserver,
         executeHolder.sessionHolder.userId,
-        executeHolder.sessionHolder.sessionId)
+        executeHolder.sessionHolder.sessionId,
+        Some(executeHolder.eventsManager).filter(_ => !interrupted))
```

Comment:
this warrants at least a code comment.
this is because you don't want to post an error after canceled?

Reply:
Correct